docs/cloud/azure/architecture.md

```diff
@@ -1 +1,937 @@
+    filesystem on Azure is likely a good solution.
+
+!!! note "Managed disks are more resilient and would still contain the data despite any Azure VM restart or loss."
+
+*Sym file (and `par.txt` for segmented databases)*
+
+:   The sym file is
+    written by the realtime database after end-of-day, when new
+    data is appended to the historical database. The HDB processes will
+    then read the sym file to reload new data. Time to read and write
+    the sym file is often marginal compared to other I/O operations.
+    Usually it is beneficial here to be able to write down to a shared
+    filesystem, thereby adding huge flexibility in the Azure Virtual
+    Network (VNet). (E.g. any other Azure VM instance can assume this
+    responsibility in a stateless fashion.)
+
+*HDB data*
+
+:   Performance of the filesystem solution will determine
+    the speed and operational latency for kdb+ to read its historical
+    (at rest) data. The solution needs to be designed to cater for good
+    query execution times for the most important business queries. These
+    may splay across many partitions or segments of data or may deeply
+    query on few or single partitions of data. The time to write a new
+    partition impacts RDB EOD work. For systems that are queried around
+    the clock the RDB write time needs to be very short.
+
+One advantage of storing your HDB within the Azure ecosystem is the
+flexibility of storage. This is usually distinct from “on-prem” storage,
+whereby you may start at one level of storage capacity and grow the
+solution to allow for dynamic capacity growth. One huge advantage of
+most Azure storage solutions (e.g persistent disks) is that disks can
+grow dynamically without the need to halt instances, this allows you to
+dynamically change resources. For example, start with small disk
+capacity and grow capacity over time.
+
+The reference architecture recommends replicating data. Either this can
+be tiered out to lower cost or lower performance object storage in Azure or
+the data can be replicated across availability zones. The latter may be
+useful if there is client-side disconnection from other time zones. You
+may consider failover of service from Europe to North America, or
+vice-versa. Kdb+ uses POSIX filesystem semantics to manage HDB
+structure directly on a POSIX-style file system stored in persistent
+storage (Azure Disk Storage _et al._). There are many solutions that offer
+full operational functionality for the POSIX interface.
+
+### Azure Blob Storage
+
+Azure Blob Storage is an object store that scales to exabytes of data.
+There are different storage classes (Premium, Hot, Cool, Archive) for
+different availability. Infrequently used data can use cheaper but
+slower storage. The kdb Insights native object store functionality allows
+users to read HDB data from Azure Blob object storage.
+
+The HDB `par.txt` file can have segment locations that are on Azure Blob
```

Comment:
Can we please use one term here.
Azure Blob Storage?
Azure Blob cloud storage?
Azure Blob Object storage?

Looks like (from the MS Azure website) it should be Azure Blob Storage, all initial capitals. Pls search and replace.